frontend_new/src/components/Res.js

```diff
@@ -19,19 +21,25 @@ function Res(props) {
     color: "#9f9f9f",
   };
 
+  const [time_delay, setDelay]= useState(1);
+
   return (
     <List>
       {!resStatus &&
         res.result.result.map(({ _id, _score, _source }) => (
```

Comment:
**[Question]** Is there a reason that we are using underscore prefixed entry mappings? Generally underscore prefixes indicate an unused field, but these are clearly used here.

Reply:
This is an issue with UIUX-30 (currently the UI main branch), not the PR. It shouldn't be addressed in this PR.